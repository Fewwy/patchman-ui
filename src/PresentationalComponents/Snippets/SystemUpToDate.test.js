import { SystemUpToDate } from './SystemUpToDate';
import { render } from '@testing-library/react';

const { asFragment } = render(<SystemUpToDate />);

describe('SystemUpToDate', () => {
    it('Should match the snapshot', () => {
        expect(asFragment()).toMatchSnapshot();
    });

    /* it('Should render CheckCircleIcon', () => {
        const iconRender = wrapper.find('EmptyStateIcon').props().icon;
        const component = iconRender();
        expect(component.type.displayName).toEqual('CheckCircleIcon');
    }); */
});
