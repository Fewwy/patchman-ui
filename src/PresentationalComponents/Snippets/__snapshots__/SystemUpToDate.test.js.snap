// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SystemUpToDate Should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="pf-l-bullseye"
  >
    <div
      class="pf-c-empty-state"
      style="padding-top: 40px;"
    >
      <div
        class="pf-c-empty-state__content"
      >
        <svg
          aria-hidden="true"
          fill="var(--pf-global--success-color--200)"
          height="3em"
          role="img"
          style="margin-bottom: 15px;"
          viewBox="0 0 512 512"
          width="3em"
        >
          <path
            d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
          />
        </svg>
        <h5
          class="pf-c-title pf-m-lg"
          data-ouia-component-id="OUIA-Generated-Title-1"
          data-ouia-component-type="PF4/Title"
          data-ouia-safe="true"
        >
          No applicable advisories
        </h5>
        <div
          class="pf-c-empty-state__body"
        >
          This system is up to date, based on package information submitted at the most recent system check-in
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
